docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -1838,3 +1845,33 @@ public class MyBean {
 === Configuration Reference
 
 include::{generated-dir}/config/quarkus-qute.adoc[leveloffset=+1, opts=optional]
+
+
+[[standalone]]
+== Qute Used as a Standalone Library
+
+Qute is primarily designed as a Quarkus extension. 
+However. it is possible to use it as a "standalone" library.
+In this case, some features are not available and some additional configuration is needed.
+
+Engine:: First of all, no managed `Engine` instance is available out of the box.
+You'll need to configure a new instance via `Engine.builder()`.
+
+Templates:: 
+* By default, no <<template-locator,template locators>> are registered, i.e. `Engine.getTemplate(String)` will not work. 
+* You can register a custom template locator or parse a template manually and put the reulst in the cache via `Engine.putTemplate(String, Template)`.
+
+Value resolvers:: 
+* No <<value-resolvers,``ValueResolver``s>> are generated automatically.
+** <<template_extension_methods,`@TemplateExtension` methods>> will not work. 
+** <<template_data,`@TemplateData`>> annotations are ignored.
+* It's recommended to register a `ReflectionValueResolver` instance via `Engine.addValueResolver(new ReflectionValueResolver())` so that Qute can access object properties and call public methods. 
```

Comment:
Maybe it would be useful to have a `addDefaultStandalone()` method or something similar to initialize with the recommended settings?

Reply:
I'm not sure it's worth it - the `ReflectionValueResolver` is the only special thing to add.

Reply:
Maybe `addDefault` should do that and we change Quarkus to not include it?

Reply:
Well, we do include this resolver in Quarkus and it's not something I'm proud of ;-). The main reason why we don't include it by default is that it's kind of a fallback solution, i.e. not a preferred nor an idiomatic way of doing the job in Qute.

Reply:
My point is that Qute standalone is not a standard way of using the engine. So it shoud not be a problem...